src/effects/native/phasereffect.cpp

```diff
@@ -173,7 +192,7 @@ void PhaserEffect::processChannel(const ChannelHandle& handle,
         right = processSample(right, oldInRight, oldOutRight, filterCoefRight, stages);
 
         // Computing output combining the original and processed sample
-        pOutput[i] = pInput[i] * (1.0 - 0.5 * depth) + left * depth * 0.5;
-        pOutput[i + 1] = pInput[i + 1] * (1.0 - 0.5 * depth) + right * depth * 0.5;
+        pOutput[i] = pInput[i] + left * depth * 0.5;
+        pOutput[i + 1] = pInput[i + 1] + right * depth * 0.5;
```

Comment:
I implemented a new way to mix the input and output signals. I think it (somewhat?) matches your proposal. The depth knob acts similar to a dry/wet knob, but because the wet signal is formed by adding two samples, it is divided by 2 before mixing with the dry signal. With the new formula, the input signal is unaffected when the depth knob is all the way down and the volume does not increase when the depth knob is all the way right. However, it is a bit difficult to hear in the middle. I'll play with this more and if it works well I'll try implementing it in #1257 for the Flanger effect too.